pkgs/unified_analytics/test/suppression_test.dart

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:file/file.dart';
+import 'package:file/memory.dart';
+import 'package:test/test.dart';
+import 'package:unified_analytics/src/enums.dart';
+import 'package:unified_analytics/unified_analytics.dart';
+
+void main() {
+  late FileSystem fs;
+  late Directory home;
+  late Analytics initializationAnalytics;
+  late Analytics analytics;
+
+  const homeDirName = 'home';
+  const initialTool = DashTool.flutterTool;
+  const measurementId = 'measurementId';
+  const apiSecret = 'apiSecret';
+  const toolsMessageVersion = 1;
+  const toolsMessage = 'toolsMessage';
+  const flutterChannel = 'flutterChannel';
+  const flutterVersion = 'flutterVersion';
+  const dartVersion = 'dartVersion';
+  const platform = DevicePlatform.macos;
+
+  final testEvent = Event.hotReloadTime(timeMs: 50);
+
+  setUp(() {
+    // Setup the filesystem with the home directory
+    final fsStyle =
```

Comment:
what style guide are you following for this codebase?

Reply:
I was originally following what we did on the flutter side but this PR a few weeks ago had a big diff
- https://github.com/dart-lang/tools/pull/108

Reply:
So what's the policy? Because I'm seeing both styles in the repo.

Reply:
I'm still following how we do things on the flutter side while fixing for any of the lints that come up, like the removing the type from the left side when assigning variables.

If you see anywhere else that needs fixing, i'm happy to fix